packages/reactivity/src/scripts/ast-analyzer.py

```
@@ -0,0 +1,380 @@
+                    {
+                        "id": block["id"],
+                        "definedVariables": [],
+                        "usedVariables": used_variables,
+                        "importedModules": [],
+                    }
+                )
+            elif block["type"] == "notebook-function":
+                input_variables = []
+                output_variables = []
+
+                if "metadata" in block and block["metadata"]:
```

Ruff (0.14.10)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)